docs/howtoguides/how_to_not_fix_related_usns.md

```diff
@@ -0,0 +1,65 @@
+# How to not fix related USNs
```

Comment:
Under what conditions would a user *not* want to fix related USNs?

Reply:
If the user wants to know beforehand which packages will actually be upgraded. By fixing related USNs, we might upgrade other packages that are not listed on the target USN. If the user has strict requirement about package version running on the machine, the user now has the possibility to fix only exactly what is listed on the target USN

Reply:
It might be a good idea to add something along those lines, I think that would help to make it clear why this is useful (and to whom)